daemon/appsettings.json

```diff
@@ -27,8 +27,9 @@
       "Frequency": "0 0 * * *"
     },
     "Updater": {
-      "Enabled": false,
-      "Frequency": "0 */12 * * *"
+      "Enabled": true,
+      "ReleaseServer": "http://localhost",
```

Comment:
This variable is not meant to be settable in an actual release, please clean it up. That'll likely get rid of the merge conflict for this file.

Frequency is messed up as well.

Reply:
Fixed. Debugging mistake.